.ruby-version

```diff
@@ -1 +1 @@
-2.3.3
+2.7.2
```

Comment:
this won't be nearly so easily done on the production server, and it'll break the ansible scripts, but i'm guessing it's a good idea to do anyway. maybe i guess @yanokwa for more thoughts if you have time.

Reply:
The upgraded deps will likely go hand in hand with this upgrade ruby version.